LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/DownlinkBasicsStationMessage.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaTools.LoRaPhysical
+{
+    using System;
+    using LoRaWan;
+
+    /// <summary>
+    /// JSON of a Downlink message for the Basic Station.
+    /// </summary>
+    public class DownlinkBasicsStationMessage
+    {
+        public string DevEui { get; }
```

Comment:
Use `DevEui` for new code?

Reply:
This is not really new code, this is the DownlinkPktFwdMessage being renamed... 
I am fine migrating this to the primitives, however that would cascade modification to a larger amount of files.